drivers/include/tm1637.h

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (C) 2024 Nico Behrens <nifrabe@outlook.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_tm1637 TM1637 display
+ * @ingroup     drivers_display
+ * @brief       Driver for the TM1637 4-digit 7-segment display
+ *
+ * @{
+ * @file
+ * @brief       Interface definition for the TM1637 4-digit 7-segment display driver
+ *
+ * @author      Nico Behrens <nifrabe@outlook.de>
+ *
+ */
+
+#ifndef TM1637_H
+#define TM1637_H
+
+#include "board.h"
+#include "periph/gpio.h"
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+/**
+ * @brief Pin configuration parameters for the tm1637 display
+ */
+typedef struct {
+    /**
+     * @brief GPIO for clock
+     */
+    gpio_t clk;
+
+    /**
+     * @brief GPIO for data input/output
+     */
+    gpio_t dio;
+} tm1637_params_t;
+
+/**
+ * @brief tm1637 driver descriptor
+ */
+typedef struct {
+    /**
+     * @brief Configuration parameters
+     *
+     */
+    tm1637_params_t params;
+} tm1637_t;
+
+/**
+ * @brief Brightness level enum for the display
+ *
+ * @note The brightness is defined as a fraction of
+ * the pulse width and must be on of the given values.
+ */
+typedef enum {
+    TM1637_PW_1_16 = 0x00,
+    TM1637_PW_2_16 = 0x01,
+    TM1637_PW_4_16 = 0x02,
+    TM1637_PW_10_16 = 0x03,
+    TM1637_PW_11_16 = 0x04,
+    TM1637_PW_12_16 = 0x05,
+    TM1637_PW_13_16 = 0x06,
+    TM1637_PW_14_16 = 0x07
+} tm1637_brightness_t;
+
+/**
+ * @brief Initializes the tm1637 device
+ *
+ * @param[out] dev device descriptor of the display
+ * @param[in] params configuration parameters
+ * @return 0 on success, error otherwise
```

Comment:
would be slightly easier to read if the params are aligned, as in

```suggestion
 * @param[out]  dev     device descriptor of the display
 * @param[in]   params  configuration parameters
 * @return 0 on success, error otherwise
```

It is apparently not explicitly mandated by the [coding convention](https://github.com/RIOT-OS/RIOT/blob/master/CODING_CONVENTIONS.md#documentation), but that's how I've seen it everywhere across RIOT until now.

Reply:
same for the rest of this file, obviously :)